.github/workflows/build-and-test-Linux.yaml

```diff
@@ -70,5 +70,10 @@ jobs:
       - name: Check for faulty input
         run: ls CPROVER/faultyInput/* || true
 
+      - uses: actions/upload-artifact@v2
+        with:
+          name: CPROVER-faultyInput
+          path: CPROVER/faultyInput/*
```

Comment:
How does one actually download these? GitHub keeps telling me that I need to have the "actions" scope, perhaps the repository permissions are set up in a way that doesn't permit non-members to do this?

Reply:
How about doing `tail -n +1 CPROVER/faultyInput/*` instead? Really simple and avoids any permissions problems.

Reply:
On the checks page (here, e.g.: https://github.com/diffblue/cbmc/pull/5916/checks there is an "artifacts" button on the top left, which allows (me) to download, I did not see any permission restrictions in the description of the action). Yes, I could also dump the content of the files, but that likely floods the output, as those are preprocessed already.

Reply:
Err, thank you, that was too easy. A first fix is in https://github.com/diffblue/cbmc/pull/5928.